app/helpers/application_helper.rb

```diff
@@ -6,4 +6,11 @@ def form_group_tag(errors, &block)
       content_tag :div, capture(&block), class: 'form-group'
     end
   end
-end
+
+  def markdown_to_html(markdown)
+    renderer = Redcarpet::Render::HTML.new
```

Comment:
Consider replacing the renderer here to be a safer one that reduces the risk from users inserting misbehaving HTML into your pages.

```diff
- renderer = Redcarpet::Render::HTML.new
+ renderer = Redcarpet::Render::Safe.new
```
The `Safe` renderer escapes HTML entered by users but still allows them to enter markdown. For example, it would reduce the likelihood of a user succesfully entering a Post that contains a `<script>` tag, like: `<script>alert('hi from attacker')</script>` which would trigger an alert popup to every user that visited that Post page.

We can discuss and demo on the next call if you're interested.